system/CodeIgniter.php

```diff
@@ -257,7 +258,12 @@ protected function initializeKint()
             Kint::$plugins = $config->plugins;
         }
 
-        Kint::$renderers[Kint::MODE_RICH] = RichRenderer::class;
+        /** @var ContentSecurityPolicy $csp */
+        $csp = Services::csp();
```

Comment:
I think the inline PHPDoc is unnecessary since it's already typehinted in `Config\Services`

Reply:
Yes, I changed my mind. We should write `@var` for local variables as little  as possible. It is a last resort.

Removed.